src/screens/App.tsx

```diff
@@ -1,23 +1,95 @@
-import { useState } from "react";
-
+import {
+  ButtonLogin,
+  ButtonCreate,
+  ButtonRegister
+} from "../components/Button/Button";
+import { FeatureTitle, FeatureDescription } from "../components/Text/Text";
 export function App() {
-  const [count, setCount] = useState(0);
-
   return (
-    <main className="flex h-screen w-full flex-col items-center justify-center gap-2 bg-black">
-      <h1 className="text-xl text-white">
-        Vite + React + Typescript + TailwindCSS | Hi
-      </h1>
-      <button
-        type="button"
-        className="bg-white p-2 text-black"
-        onClick={() => {
-          setCount((count) => count + 1);
-        }}
-      >
-        Increase
-      </button>
-      <p className="text-white">Count: {count}</p>
+    <main className="h-screen w-full">
+      <div className="min-h-1/2 bg-blue-600 pb-16">
+        <div className=" mx-auto max-w-screen-2xl">
```

Comment:
1. Use the `<nav></nav>` tags for your navbar. 
2. The navbar should be a separated component since it will have its own state to show different options according to the user's session. 
3. The new `Navbar` component should appear in all the pages.